exp/tools/serpapi/internal/serpapiClient.go

```diff
@@ -0,0 +1,109 @@
+package internal
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+var ErrMissingToken = errors.New("missing the OpenAI API key, set it in the OPENAI_API_KEY environment variable")
```

Comment:
prefer a `var (` expression

Reply:
there's a copy and paste error, it's not the openAPI key that's missing ;)

Reply:
Sorry i fixed it in the Tools PR, forgot to change it here #19